Documentation/install.md

```diff
@@ -4,25 +4,18 @@
 
 Choose one way to install inspektor-gadget.
 
-### Quick installation
+### Stable version
 
 ```
-$ curl -s -L --output inspektor-gadget.zip https://github.com/kinvolk/inspektor-gadget/suites/333471026/artifacts/477863
-$ unzip inspektor-gadget.zip
-$ chmod +x inspektor-gadget/inspektor-gadget
+$ wget https://github.com/kinvolk/inspektor-gadget/releases/download/v0.1.0-alpha.3/inspektor-gadget.tar.gz
+$ tar xvf inspektor-gadget.tar.gz
 $ sudo cp inspektor-gadget/inspektor-gadget /usr/local/bin/kubectl-gadget
+$ kubectl gadget version
 ```
 
-Check installation:
-
-```
-$ inspektor-gadget version
-v0.1.0-alpha.2-12-gb9fd574
-```
+You can find other releases on [releases](https://github.com/kinvolk/inspektor-gadget/releases).
 
-### From a release
-
-Follow instructions from the [releases](https://github.com/kinvolk/inspektor-gadget/releases).
+### Latest version
```

Comment:
this section seems empty (or the next line should have 4 `#`).

Reply:
Right, there was a missing `#` on the next line.